members/W000779.yaml

```diff
@@ -3,131 +3,229 @@ contact_form:
   method: POST
   action: ""
   steps:
-    - visit: "http://www.wyden.senate.gov/contact"
+    - visit: "https://www.wyden.senate.gov/contact/email-ron"
+    - click_on:
+      - selector: div .btn-block h1
```

Comment:
This selector is not unique to the element we want: https://gyazo.com/d61802f4baa3dbe30f413e57ca79b34c .